impress-org/givewp
# Totals block now has feature parity with the Give Totals shortcode

Resolves #5260 

## Description
This PR strips the Milestone block so that it has feature parity with the Give Totals shortcode. Additionally, this PR modifies all references to the Milestones feature, so that it is instead called "Totals" moving forward in the src directory. The PR also improves the default message, so that it showcases all available dynamic tags. 

After some more local testing, I am again hesitant to remove the metric selection feature here. Because we are not actually modifying the Give Totals shortcode functionality in any way, it feels fine to include this minor improvement over the original implementation. For this PR, I have opted to leave it in place.

## Affects
This PR touches almost every file associated with the Milestone feature. Specifically, it changes references to how the Milestone block was registered, so that it is instead referred to as the Totals block. Additionally, it removes key features of Milestones like featured image, title and deadline to achieve feature parity with the Give Totals shortcode. 

## Visuals
<img width="646" alt="Screen Shot 2020-09-15 at 12 50 48 PM" src="https://user-images.githubusercontent.com/5186078/93241274-23038180-f753-11ea-9c71-46d6560dd664.png">

<img width="275" alt="Screen Shot 2020-09-15 at 12 51 49 PM" src="https://user-images.githubusercontent.com/5186078/93241287-25fe7200-f753-11ea-8121-17e0cabf36b0.png">

## Pre-review Checklist

## Testing Instructions
1. Create a new post or page
2. Add the Totals block
3. Test out each setting, check that it appears as expected both in the editor and on the frontend
